BrainPortal/app/controllers/groups_controller.rb

```diff
@@ -222,7 +222,7 @@ def unregister
   # DELETE /groups/1.xml
   # DELETE /groups/1.json
   def destroy  #:nodoc:
-    @group = current_user.available_groups.find(params[:id])
+    @group = current_user.assignable_groups.find(params[:id])
```

Comment:
Really ? I am pretty sure you want to put `modifiable_groups` here. You probably do not want to allow people with assignable right to delete a group.

Reply:
Well, it happens to be exactly equivalent to what was in CBRAIN before... and yes I feel that's a CBRAIN bug maybe

Reply:
I'll set it to modifiable_groups